src/test/java/com/github/cukespace/arquillian/asciidoctor/AdocExtensionTest.java

```diff
@@ -0,0 +1,125 @@
+package com.github.cukespace.arquillian.asciidoctor;
+
+/**
+ * Created by pestano on 03/09/16.
+ */
+
+import org.jboss.arquillian.config.descriptor.api.ArquillianDescriptor;
+import org.jboss.arquillian.config.descriptor.api.ContainerDef;
+import org.jboss.arquillian.config.descriptor.impl.ArquillianDescriptorImpl;
+import org.jboss.arquillian.container.spi.Container;
+import org.jboss.arquillian.container.spi.client.container.DeployableContainer;
+import org.jboss.arquillian.core.api.annotation.ApplicationScoped;
+import org.jboss.arquillian.core.api.event.ManagerStarted;
+import org.jboss.arquillian.core.api.event.ManagerStopping;
+import org.jboss.arquillian.core.spi.ServiceLoader;
+import org.jboss.arquillian.core.test.AbstractManagerTestBase;
+import org.junit.*;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.contentOf;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.when;
+
+
+@RunWith(MockitoJUnitRunner.class)
```

Comment:
I'd like to have no mock at all in tests, if you want lightweight tests feel free to call directly the @Observes method in a test but mocks just brings ways to pass tests and break the actual feature there IMHO